R/compose-input-for-stan.R

```diff
@@ -233,7 +243,9 @@ get_sufficient_statistics_as_list_of_arrays <- function(
 #' @param lapse_rate,mu_0,Sigma_0 Optionally, lapse rate, prior expected category means (mu_0) and/or prior expected
 #' category covariance matrices (Sigma_0) for all categories. Lapse rate should be a number between 0 and 1. For mu_0
 #' and Sigma_0, each should be a list, with each element being the expected mean/covariance matrix for a specific
-#' category prior to updating. Elements of mu_0 and Sigma_0 should be ordered in the same order as the levels of the
+#' category \emph{including perceptual noise} prior to updating (including noise is necessary since the stancode for the
```

Comment:
does this mean one does not have to specify adding noise variance in the Sigma_0 argument when fitting the IA? Great.